src/renderer/components/+user-management/user-management.tsx

```diff
@@ -13,29 +13,37 @@ import { isAllowedResource } from "../../../common/rbac";
 @observer
 export class UserManagement extends React.Component {
   static get tabRoutes() {
-    const tabRoutes: TabLayoutRoute[] = [];
     const query = namespaceUrlParam.toObjectParam();
+    const tabRoutes: TabLayoutRoute[] = [];
 
-    tabRoutes.push(
-      {
+    if (isAllowedResource("serviceaccount")) {
+      tabRoutes.push({
         title: "Service Accounts",
         component: ServiceAccounts,
         url: serviceAccountsURL({ query }),
         routePath: serviceAccountsRoute.path.toString(),
-      },
-      {
+      });
+    }
+
+    if (isAllowedResource("rolebinding") || isAllowedResource("clusterrolebinding")) {
+      // TODO: seperate out these two pages
+      tabRoutes.push({
         title: "Role Bindings",
         component: RoleBindings,
         url: roleBindingsURL({ query }),
         routePath: roleBindingsRoute.path.toString(),
-      },
-      {
+      });
+    }
+
+    if (isAllowedResource("role") || isAllowedResource("clusterrole")) {
+      // TODO: seperate out these two pages
```

Comment:
Why those pages should be separated?

Reply:
It relates to #2199. Namely, they are different resources with slightly different semantics so they should be on different pages.